### PR TITLE
TG-126 : implementation of react hook forms in the admins entity.

### PR DESCRIPTION
**Pull request by javierlazzarini**
JOI validations.
React hook forms implementation.

### CODE REVIEW

File: src/Components/Admins/index.js [hunk 1]
```diff
@@ -1,33 +1,66 @@
...
+    email: Joi.string()
+      .email({ tlds: { allow: false } })
+      .min(7)
+      .required(),
+    password: Joi.string()
+      .min(8)
+      .required()
+      .regex(/(?!^[0-9]*$)(?!^[a-zA-Z]*$)^([a-zA-Z0-9]{8,25})$/),
+    active: Joi.boolean().required()
+  });
```
Comment (martinalbrnz) [inline src/Components/Admins/index.js:43]: Here you can use 
```
projectManager: Joi.string()
      .min(3)
      .max(30)
      .regex(/^([ \u00c0-\u01ffa-zA-Z'-])+$/)
      .messages({
        'string.min': 'Project manager name must contain 3 or more letters',
        'string.max': 'Project manager name must contain 30 or less letters',
        'string.empty': 'This field is required',
        'string.pattern.base': 'Name is not valid',
        'string.regex': 'Project manager name is not valid'
      })
      ```
      to customize error messages

Comment (nicolascostanza) [reply src/Components/Admins/index.js:43]: x2, mensajes específicos darían una mejor experiencia de usuario

File: src/Components/Admins/index.js [hunk 2]
```diff
@@ -106,17 +125,15 @@ function Admins() {
...
     }
     setMethod('');
-    resetFields();
     handleCloseAdd(false);
   };
 
-  const onSubmit = (e) => {
-    e.preventDefault();
+  const onSubmit = (data) => {
+    console.log(data);
```
Comment (nicolascostanza) [inline src/Components/Admins/index.js:132]: este console.log no va

**Review by nicolascostanza — DISMISSED** [review]

**Comment by Clarosluciano** [comment]
# Test result: 
Functionalities (CRUD): fail
Validations: fail

# Bugs found:
1. The validations do not correspond to the input in question:
image
**Expected behavior:** First name and Last name validations should be "only letters".

2. The submit button does nothing:
image
**Expected behavior:** The submit button should send the request to the API and the user should receive a response.

3. The reset form button does nothing. The button layout is weird. The modal is small:
image
**Expected behavior:** The reset form button should do its job. The layout of the buttons should be arranged. The modal should be a bit bigger.

4. The first time the add or edit button are clicked, an extra loarder appears at the top of the screen:
image
image
**Expected behavior:** Only one loader should appear on the screen.

**Review by GinaSchP — APPROVED** [review]

**Review by mpueblas — APPROVED** [review]
yo lo veo very beautiful

**Review by Clarosluciano — APPROVED** [review]
Gj!

**Review by nicolascostanza — APPROVED** [review]

**Pull request merged**